src/authrite.js

```diff
@@ -156,14 +156,25 @@ class Authrite {
       returnType: 'wif'
     })
     // Make sure the fetchConfig body is formatted to the correct content type
+    // TODO: Check fetchConfig.headers['Content-Type'] to support other data types
+    if (!fetchConfig.headers) {
+      fetchConfig.headers = {}
+    }
     let dataToSign
-    if (fetchConfig.body) {
-      fetchConfig.body = typeof (fetchConfig.body) === 'string' ? fetchConfig.body : JSON.stringify(fetchConfig.body)
-      dataToSign = fetchConfig.body
-      // TODO: Check fetchConfig.headers['Content-Type'] to support other data types
-      if (!fetchConfig.headers) {
-        fetchConfig.headers = {}
+    // Check if we should sign the requestURL instead of the body
+    // The fetch API POST request method defaults to 'GET'
+    fetchConfig.method ??= 'GET'
```

Comment:
```suggestion
    if (!fetchConfig.method) {
      fetchConfig.method = 'GET'
    }
```
Unfortunately, this is too new and some Google AppEngine things were breaking when we tried it. One day Google, ONE DAY.